### ISSUE TITLE
What is the difference between considering and ignoring bias?

### ISSUE THREAD

**Issue by wjNam** [albermax/innvestigate#117]
Hi :)

I'm studying for explaining DNN and have some questions for considering bias and batchnormalization in source code.

What is the exact difference between keeping bias and ignoring bias?

Is it the process of subtracting the effect of bias from the input relevance?

or is it process for only considering the activation of neuron before and after applying bias?

And for explaining models with batchnormalization (resnet, densenet), is it fully implemented? or still processing?

It seems that the heatmaps provide better explanations when algorithm ignores the batchnormalization reverse process and just propagates input relevance to next layer.

Thank you for reading.

From Nam

**Comment by albermax** [comment]
Hi Nam,

thank you for your questions! Could you please clarify of which methods you are speaking about?

Cheers,
Max

**Comment by wjNam** [comment]
Hi :) Thank you for your answering.

My question is for the methods which are based on relevance propagation, such as LRP, LRP-alphabeta, LRP-presetA and B.

I know that DTD is not support the batch normalization.

Thank you.

**Comment by albermax** [comment]
Hi Nam,

then I would like to forward this question to @sebastian-lapuschkin 

Cheers,
Max

**Comment by sebastian-lapuschkin** [comment]
I will respond asap, as soon as there is some time (ie next week).

**Comment by wjNam** [comment]
Thank you. I will waiting for your answering.

And Here, i have a few questions more.

I understand that LRP-alphabeta separates positive/negative relevance according to its weights.

But i can't understand why positive relevance should be xpos*wpos + xneg*wneg.

And i think that Xs (which is input layer for relevance propagation) is always over zero, because it already passed activation function (Relu).

Is there any cases that input of hidden layers is below zero?

I'm confusing...

Sorry for bothering you.

Have a nice day:)

**Comment by albermax** [comment]
Hi Woo-Joeung,

to your last question: the input is only greater zero if the previous layer was a ReLU activation. The code is more complex to cover also other cases.

Cheers,
Max
